gameserver/migrations/0002_auto_20211208_0457.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 3.2.8 on 2021-12-08 04:57
```

Comment:
probably will have to remake this migration to merge with the permission change migrations in main branch

Reply:
migration re-made as 0005